app/views/formatted-object.html

```diff
@@ -0,0 +1,16 @@
+<div class="formatted-object">
+  <div class="formatted-object__title"><a ng-click="toggleExpand()"><i class="fa fa-chevron-right" ng-class="isExpanded ? 'expanded-icon' : ''"></i> Object</a></div>
```

Comment:
how do you feel about using http://fontawesome.io/icon/caret-right/ (because we also using in the nodes view )?